app/services/mock_benefit_check_service.rb

```diff
@@ -0,0 +1,47 @@
+class MockBenefitCheckService
+  KNOWN = {
+    'SMITH' => { nino: 'ZZ123459A', dob: '11-Jan-99' },
+    'JONES' => { nino: 'ZZ123458A', dob: '1-Jun-80' },
+    'BLOGGS' => { nino: 'ZZ123457A', dob: '4-Jan-90' },
+    'WRINKLE' => { nino: 'ZZ010150A', dob: '01-Jan-50' },
+    'WALKER' => { nino: 'JA293483A', dob: '10-Jan-80' }, # Used in cucumber tests and specs
```

Comment:
ZZ is an invalid prefix, so won't pass our NINO validation. Can we change them to AB or something?